### DISCUSSION TITLE
Question about tfcmt

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [suzuki-shunsuke/tfcmt#247] [Announcements]
If you have any question about the usage of tfcmt, please ask it freely.

**Comment by vijyantg** [comment]
<img width="1009" alt="image" src="https://user-images.githubusercontent.com/81227957/231946272-90dd1a20-c33c-42de-9fec-53ae1434056d.png">
I am getting error like github_token not found 
<img width="852" alt="image" src="https://user-images.githubusercontent.com/81227957/231946380-6b5dd3e6-b39a-4bed-98cc-26d2ab95e99f.png">

**Reply by suzuki-shunsuke** [reply]
Hmm. It looks strange. 🤔
The error means the environment variable `GITHUB_TOKEN` isn't set or empty.

https://github.com/suzuki-shunsuke/tfcmt/blob/af0cef9cd3d7cc052eea7cd1a0306463d3e87160/pkg/notifier/github/client.go#L78-L83

BTW, `export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}` is unnecessary because GITHUB_TOKEN is set by `env`.

```yaml
env:
  GITHUB_TOKEN: ${{ github.token }}
```

**Reply by suzuki-shunsuke** [reply]
BTW, please paste code instead of screenshot from the next time because screenshot can't be copy and paste.

**Comment by vijyantg** [comment]
thanks !!
